projects/go-lib/src/lib/components/go-side-nav/go-side-nav/go-side-nav.component.scss

```diff
@@ -28,3 +27,39 @@
     width: $side-nav-width--collapsed;
   }
 }
+
+.go-side-nav__nav-items {
+  overflow-y: auto;
+}
+
+.go-side-nav__app {
+  margin-top: auto;
```

Comment:
Can we remove this in favor of the `flex: 1;` being added to the `go-side-nav__nav-items`.

Reply:
yes, removed